tests/unit/test_business_logic.py

```diff
@@ -1343,11 +1344,13 @@ def test_release_request_add_same_file(bll):
     assert len(release_request.filegroups["default"].files) == 1
 
 
-def _get_current_file_reviews(bll, release_request, path, author):
+def _get_request_file(bll, release_request, path):
     """Syntactic sugar to make the tests a little more readable"""
     # refresh
-    release_request = bll.get_release_request(release_request.id, author)
-    return release_request.get_request_file_from_output_path(path).reviews
+    release_request = bll.get_release_request(
```

Comment:
There is `factories.refresh_release_request()` which will do the user-creating if necessary for you